i18n/hr.yaml

```diff
@@ -0,0 +1,81 @@
+- id: sitemap
+  translation: Mapa stranice
+
+- id: tags
+  translation: Oznake
+
+- id: categories
+  translation: Kategorije
+
+- id: social
+  translation: Društvene mreže
+
+- id: links
+  translation: Poveznice
+
+- id: rss_feed
+  translation: RSS unosi
+
+- id: tag
+  translation: Oznaka
+
+- id: category
+  translation: Kategorija
+
+- id: archive_of_tag
+  translation: Arhive izlistane po oznakama
+
+- id: archive_of_category
+  translation: Arhive izlistane po kategorijama
+
+- id: archive
+  translation: Arhive
+
+- id: list_of_tags
+  translation: Popis oznaka
+
+- id: list_of_categories
+  translation: Popis kategorija
+
+- id: more_post
+  translation: Više objava
+
+- id: articles_tagged
+  translation: označeni članci
+
+- id: article_tagged
+  translation: označeni članak
+
+- id: error_404
+  translation: 'Greška 404 - stranica nije pronađena'
+
+- id: in_category
+  translation: U kategorijama
+
+
+- id: latest_posts
+  translation: Najnovije objave
+
+- id: page
+  translation: Stranica
+
+- id: previous
+  translation: Prethodna
+
+- id: next
+  translation: Slijedeća
+
+- id: author_by
+  translation: autor
```

Comment:
I assume this is a translation workaround for a troublesome "by". Is that a correct assumption?

Reply:
Nearly completely ignorant to the grammatical nuances of Croatian, so bear with my comment.

Reply:
Can you give me an example of a whole sentence so I can check the translation?

Reply:
Not 100% sure. Perhaps it is in this string "By Ursula Kallio / 2019-12-17 / In categories Posts" at https://infobits.org/en/asynchronous-online-teaching/?

I am not sure if the initially is being forced as an uppercase letter or not without more investigation.

Reply:
Yep, it’s here: https://github.com/shaform/hugo-theme-den/blob/master/i18n/en.yaml#L69
In English, the term is simply "By".

Reply:
In this case, the translation is correct.